components/channel_notifications_modal/channel_notifications_modal.jsx

```diff
@@ -201,29 +205,33 @@ export default class ChannelNotificationsModal extends React.Component {
                                     onUpdateSection={this.handleUpdateMarkUnreadSection}
                                     serverError={serverError}
                                 />
-                                <div className='divider-light'/>
-                                <NotificationSection
-                                    section={NotificationSections.DESKTOP}
-                                    expand={activeSection === NotificationSections.DESKTOP}
-                                    memberNotificationLevel={desktopNotifyLevel}
-                                    globalNotificationLevel={currentUser.notify_props ? currentUser.notify_props.desktop : NotificationLevels.ALL}
-                                    onChange={this.handleUpdateDesktopNotifyLevel}
-                                    onSubmit={this.handleSubmitDesktopNotifyLevel}
-                                    onUpdateSection={this.handleUpdateDesktopSection}
-                                    serverError={serverError}
-                                />
-                                <div className='divider-light'/>
-                                {sendPushNotifications &&
-                                <NotificationSection
-                                    section={NotificationSections.PUSH}
-                                    expand={activeSection === NotificationSections.PUSH}
-                                    memberNotificationLevel={pushNotifyLevel}
-                                    globalNotificationLevel={currentUser.notify_props ? currentUser.notify_props.push : NotificationLevels.ALL}
-                                    onChange={this.handleUpdatePushNotificationLevel}
-                                    onSubmit={this.handleSubmitPushNotificationLevel}
-                                    onUpdateSection={this.handleUpdatePushSection}
-                                    serverError={serverError}
-                                />
+                                {!isChannelMuted(channelMember) &&
+                                <div>
```

Comment:
nit: Do we usually indent these blocks?

Reply:
Never have, but I could if someone is 1/5

Reply:
I usually do indent but I'm always tempted to do it like this since I want to see sibling elements (in this case `div`) to be aligned.
0/5 since we are not consistent in this pattern (atleast in webapp).

Reply:
@enahum Feel free to merge.